configuration/openmrs/apps/registration/registrationCardLayout/print.html

```diff
@@ -0,0 +1,102 @@
+<link rel="stylesheet" href="/bahmni_config/openmrs/apps/registration/registrationCardLayout/css/print.css"/>
+
+<p>
+    <img style="float: left; width: 85mm; height: 15;" src="/bahmni_config/openmrs/apps/registration/registrationCardLayout/images/dsjmc-rect-logo.png" alt=""  />
+    <img ng-src="{{patient.image}}" style="float: right; width: 30mm; height: 30mm; "
+    onerror="this.onerror=null;this.src='../images/blank.jpeg';" />
+</p>
+
```

Comment:
@chinzou is there any particular reason for having these empy lines here?

Reply:
@rbuisson , no there is no need to the empty lines